app/Theme/Plugins.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Dxw\MyTheme\Theme;
+
+use Dxw\Iguana\Registerable;
+
+class Plugins implements Registerable
+{
+    protected $required;
+    protected $abspath;
```

Comment:
What does abspath mean? Is there a name we could give it that is easier to understand and/or specific to this use case?